### PR TITLE
Update icons for legibility with dark themes and adjustments for artwork guidelines

### LINKED ISSUE #34: Legibility on dark themes
@edwardvmills I'd be happy to overhaul the icons of this WB to be consistent with FreeCAD Artwork Guidelines and to be legible with dark themes:
image

Before I start one question: You have a .svg and .png version of the icons. Is the .png used anywhere? Also the same icons seem to be duplicated with a different name, could they be unified to use the same .svg source?

image

### PR DESCRIPTION
**Pull request by maxwxyz**
Fixes https://github.com/edwardvmills/Silk/issues/34
Update icons for legibility with dark themes and aligned them with FreeCAD artwork guidelines.

Previously:
320026236-d3df7ab6-b13a-4e9a-beed-e5e5ded3e540

PR:
image

@edwardvmills FYI

### CODE REVIEW

**Comment by maxwxyz** [comment]
@edwardvmills ping

**Comment by maxwxyz** [comment]
@edwardvmills could you click the merge button or does it need adjustment?

**Comment by edwardvmills** [comment]
TLDR: i may temporarily accept, but I will want to revisit with separate icons for light and dark modes eventually.

Could you please drop your files in Silk's icon folder, load the WB in FreeCAD, and send me some screenshots of how the toolbars end up looking in both dark and light mode (screenshot a large portion of the application please)?

**Comment by maxwxyz** [comment]
@edwardvmills sure.

Currently:
Light
grafik
Dark
grafik


PR:
Light
grafik

Dark
grafik

**Comment by edwardvmills** [comment]
Thank you Max. there's a a few object drawing order issues i'll need to fix, as they were present before, but not as noticeable.
